tests/yam/agama/agama_reboot.pm

```diff
@@ -0,0 +1,19 @@
+## Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Use single module that handle reboot for all architectures
+# integration test from GitHub.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use strict;
+use warnings;
+use base "installbasetest";
+use testapi qw(get_var);
+use Utils::Architectures;
+use utils 'reconnect_mgmt_console';
+
+sub run {
+    reconnect_mgmt_console if is_s390x || (get_var('AGAMA_TEST') eq 'test_full_disk_encryption');
```

Comment:
In fact, here should add reconnect for remote workers. While someone added the reconnect for powerVM in grub_test, this will cause duplicate modules if added here.
And the condition is wrong, should be is_s390x or test_full_disk_encryption on remote worker. I don't like the complicated condition, while it should match such condition.

Reply:
> In fact, here should add reconnect for remote workers. While someone added the reconnect for powerVM in grub_test, this will cause duplicate modules if added here. And the condition is wrong, should be is_s390x or test_full_disk_encryption on remote worker. I don't like the complicated condition, while it should match such condition.

Changed and thanks, I'd think of other way to address your concerns.